src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/InternalConnectionContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Data.SqlClientX.Handlers
+{
+    /// <summary>
+    /// Placeholder context class needed for SqlConnector
+    /// </summary>
+    internal class InternalConnectionContext
```

Comment:
If this is needed for SqlConnection, shouldn't it be added by the SqlConnector?
I'm confused why this is necessary.

Reply:
Not needed IMO too.

Reply:
Would not be needed after the next PR.